doc/pachub/pachub_getting_started.md

```diff
@@ -162,16 +166,17 @@ To connect to your cluster, complete the following steps:
 1. Log in to your Pachyderm cluster:
 
    1. Go to the Pachyderm Hub UI.
-   1. In the list of clusters, click **Dashboard**.
-      Pachyderm Hub opens the Dashboard in a new window.
-   1. Click **Settings**.
-   1. Click **Generate New One-Time Password Now**.
+   1. In the list of clusters, click **Details**.
+   1. Click **>**.
```

Comment:
one minor part of the PR connor is working on changes this. It's a very easy update for me to make myself if/when it lands. This is fine for now